src/app/ngrx/actions/action.types.ts

```diff
@@ -1,39 +1,35 @@
-export const ActionTypes = {
-  CMS_ACCOUNT: 'CMS_ACCOUNT',
-  CMS_ACCOUNT_SUCCESS: 'CMS_ACCOUNT_SUCCESS',
-  CMS_ACCOUNT_FAILURE: 'CMS_ACCOUNT_FAILURE',
-  CMS_PODCASTS: 'CMS_PODCASTS',
-  CMS_PODCASTS_SUCCESS: 'CMS_PODCASTS_SUCCESS',
-  CMS_PODCASTS_FAILURE: 'CMS_PODCASTS_FAILURE',
-  CMS_RECENT_EPISODE: 'CMS_RECENT_EPISODE',
-  CMS_RECENT_EPISODE_SUCCESS: 'CMS_RECENT_EPISODE_SUCCESS',
-  CMS_RECENT_EPISODE_FAILURE: 'CMS_RECENT_EPISODE_FAILURE',
-  CMS_PODCAST_EPISODE_PAGE: 'CMS_PODCAST_EPISODE_PAGE',
-  CMS_PODCAST_EPISODE_PAGE_SUCCESS: 'CMS_PODCAST_EPISODE_PAGE_SUCCESS',
-  CMS_PODCAST_EPISODE_PAGE_FAILURE: 'CMS_PODCAST_EPISODE_PAGE_FAILURE',
-  CASTLE_PODCAST_METRICS_LOAD: 'CASTLE_PODCAST_METRICS_LOAD',
-  CASTLE_PODCAST_METRICS_SUCCESS: 'CASTLE_PODCAST_METRICS_SUCCESS',
-  CASTLE_PODCAST_METRICS_FAILURE: 'CASTLE_PODCAST_METRICS_FAILURE',
-  CASTLE_PODCAST_PERFORMANCE_METRICS_LOAD: 'CASTLE_PODCAST_PERFORMANCE_METRICS_LOAD',
-  CASTLE_PODCAST_PERFORMANCE_METRICS_SUCCESS: 'CASTLE_PODCAST_PERFORMANCE_METRICS_SUCCESS',
-  CASTLE_PODCAST_PERFORMANCE_METRICS_FAILURE: 'CASTLE_PODCAST_PERFORMANCE_METRICS_FAILURE',
-  CASTLE_EPISODE_METRICS_LOAD: 'CASTLE_EPISODE_METRICS_LOAD',
-  CASTLE_EPISODE_METRICS_SUCCESS: 'CASTLE_EPISODE_METRICS_SUCCESS',
-  CASTLE_EPISODE_METRICS_FAILURE: 'CASTLE_EPISODE_METRICS_FAILURE',
-  CASTLE_EPISODE_PERFORMANCE_METRICS_LOAD: 'CASTLE_EPISODE_PERFORMANCE_METRICS_LOAD',
-  CASTLE_EPISODE_PERFORMANCE_METRICS_SUCCESS: 'CASTLE_EPISODE_PERFORMANCE_METRICS_SUCCESS',
-  CASTLE_EPISODE_PERFORMANCE_METRICS_FAILURE: 'CASTLE_EPISODE_PERFORMANCE_METRICS_FAILURE',
-  GOOGLE_ANALYTICS_EVENT: 'GOOGLE_ANALYTICS_EVENT',
-  CUSTOM_ROUTER_NAVIGATION: 'CUSTOM_ROUTER_NAVIGATION',
-  ROUTE_SERIES: 'ROUTE_SERIES',
-  ROUTE_PODCAST_CHARTED: 'ROUTE_PODCAST_CHARTED',
-  ROUTE_EPISODES_CHARTED: 'ROUTE_EPISODES_CHARTED',
-  ROUTE_SINGLE_EPISODE_CHARTED: 'ROUTE_SINGLE_EPISODE_CHARTED',
-  ROUTE_TOGGLE_EPISODE_CHARTED: 'ROUTE_TOGGLE_EPISODE_CHARTED',
-  ROUTE_EPISODE_PAGE: 'ROUTE_EPISODE_PAGE',
-  ROUTE_CHART_TYPE: 'ROUTE_CHART_TYPE',
-  ROUTE_INTERVAL: 'ROUTE_INTERVAL',
-  ROUTE_STANDARD_RANGE: 'ROUTE_STANDARD_RANGE',
-  ROUTE_ADVANCED_RANGE: 'ROUTE_ADVANCED_RANGE',
-  ROUTE_METRICS_TYPE: 'ROUTE_METRICS_TYPE'
-};
+export enum ActionTypes {
```

Comment:
NgRx is now recommending string enums as the standard for action types. Changing the string values to `[source] ACTION` makes them more readable in the redux dev tools.